nx-X11/lib/X11/XlibInt.c

```diff
@@ -3759,36 +4178,6 @@ Screen *_XScreenOfWindow (dpy, w)
     return NULL;
 }
 
-/*
- * WARNING: This implementation's pre-conditions and post-conditions
- * must remain compatible with the old macro-based implementations of
- * GetReq, GetReqExtra, GetResReq, and GetEmptyReq. The portions of the
- * Display structure affected by those macros are part of libX11's
- * ABI.
- */
-void *_XGetRequest(Display *dpy, CARD8 type, size_t len)
```

Comment:
Don't remove this. It will cause a regression. You yourself added this some weeks ago to fix crashes in libXrender IIRC.